### PR TITLE
Implement hybrid function `bindTestRun` method  (#241)

### PR DESCRIPTION
**Pull request by inikulin**
\cc @helen-dikareva @AlexanderMoskovkin 
@VasilyStrelyaev please, check strings

### CODE REVIEW

File: test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js [hunk 1]
```diff
@@ -56,3 +56,37 @@ test('Generator in Hybrid', async () => {
         yield 1;
     });
 });
+
+test('Bind Hybrid function', async t => {
+    const fs               = require('fs');
+    const boundGetLocation = getLocation.bindTestRun(t);
+
+    // NOTE: binding should not modify original Hybrid
+    // function bound test run, so it should throw.
```
Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js]:
```
    // NOTE: binding should not modify the test run 
    // bound to the Hybrid function originally, so it should throw an error.
```

Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js]: Did I get it right?

**Comment by VasilyStrelyaev** [comment]
\r-

File: test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js (hunk 1 above)
Comment (inikulin) [inline test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js]: Nope, but my version was incorrect as well. The meaning is that binding doesn't change original function's binding, but creates new bound function instead.

Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js]: So, we can say it does not modify the function itself, can't we?
```
    // NOTE: binding does not modify the original function,
    // but creates a new bound function instead, so here we should throw an error.
```

Comment (inikulin) [inline test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js]: Yep, thanks

**Comment by inikulin** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
lgtm

**Pull request merged**